swagger.json

```diff
@@ -1503,11 +1503,16 @@
                     "type": "number"
                 },
                 "network": {
-                    "type": "string"
+                    "type": "object",
+                    "additionalProperties": {
+                       "type": "object"
```

Comment:
I don't this is correct.

Reply:
With this change, The type for attributes generated is 
```
@SerializedName("attributes")
  private Map<String, Object> attributes = null;
```
I feel this is a better way to read and manage stuff at the client side.